pacemaker.spec.in

```diff
@@ -248,6 +248,8 @@ BuildRequires: inkscape asciidoc publican
 %endif
 %endif
 
+Provides:      pcmk-cluster-manager%{?_isa} = %{version}-%{release}
```

Comment:
On the bright side, it doesn't spoil `pacemaker-*` namespace meaning
that we can see arising systemic, legible dichotomy as a feature:
- `pacemaker*` real non-virtual packages carrying some actual bits
- `pcmk-*` virtual/alias packages

And might serve as flag put in the `pcmk-*` package naming territory,
so hopefully it will get considered should anyone want to pick that
name for any later piece of software to come.